### PR TITLE
changelog: Fix erroneous entries

### PR DESCRIPTION
**Pull request by nfuden**

### CODE REVIEW

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -4,44 +4,40 @@ on:
...
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
       with:
         fetch-depth: 0
-    - uses: google-github-actions/setup-gcloud@v0.2.1
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: 3.8
+    - uses: google-github-actions/setup-gcloud@a48b55b3b0eeaf77b6e1384aab737fbefe2085ac
```
Comment (jenshu) [inline .github/workflows/pull_request.yaml:26]: why this pin?

File: .github/workflows/pull_request.yaml [hunk 2]
```diff
@@ -4,44 +4,40 @@ on:
   push:
     branches:
     - 'master'
+    - 'main'
```
Comment (jenshu) [inline .github/workflows/pull_request.yaml:7]: why both master and main? can master be removed?

Comment (nfuden) [reply .github/workflows/pull_request.yaml:7]: File copied directly from main branch

File: .github/workflows/pull_request.yaml (hunk 1 above)
Comment (nfuden) [reply .github/workflows/pull_request.yaml:26]: Unsure copied from main and trusted prior art

Comment (nfuden) [reply .github/workflows/pull_request.yaml:26]: https://github.com/solo-io/go-utils/commit/ab76a9f5217cd95d8b8c1eca87e9e7087864aba0

File: .github/workflows/pull_request.yaml (hunk 2 above)
Comment (nfuden) [reply .github/workflows/pull_request.yaml:6]: eh removed

**Review by jenshu — APPROVED** [review]

**Pull request merged**